src/common-components/assert.response.unsupportedResource.xml

```diff
@@ -6,43 +6,44 @@
 <nts:component xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript">
     <action>
         <assert nts:stopTestOnFail="true">
-            <description value="Confirm that the server responds with a 404 not found"/>
+            <description value="Confirm that the returned HTTP status is 404 (Not Found)."/>
+            <direction value="response"/>
             <operator value="equals"/>
             <responseCode value="404"/>
         </assert>
     </action>
     <action>
         <assert nts:stopTestOnFail="true">
-            <description value="Confirm that the server returns a single OperationOutcome resource"/>
+            <description value="Confirm that the returned resource type is OperationOutcome."/>
             <direction value="response"/>
             <resource value="OperationOutcome"/>
         </assert>
     </action>
     <nts:profile id="OperationOutcome-profile" value="http://hl7.org/fhir/StructureDefinition/OperationOutcome"/>
     <action>
         <assert>
-            <description value="Validate the OperationOutcome resource against the base definition"/>
+            <description value="Confirm that the returned OperationOutcome conforms to the base FHIR specification."/>
             <direction value="response"/>
             <validateProfileId value="OperationOutcome-profile"/>
```

Comment:
As the profile id is used here, the `nts:profile` element should also be placed within this `nts:component`. And following this, it should be removed from the individual NTS-files

Reply:
The output in dev shouldn't change because of this

Reply:
Appartently this common-component isn't included correctly? https://touchstone.aegis.net/touchstone/scriptexecution?exec=202211100843078086862099&qn=/FHIRSandbox/Nictiz/dev/FHIR3-0-2-MM202001-Cert/PDFA-3-0/XIS-Server-NoManifest/medmij-pdfa-xis-2-2-serve-2-documentmanifest-xml#WAITING_OPER is missing these checks completely